Modules/rat_rescue.py

```diff
@@ -675,7 +672,7 @@ def title(self, value: str or None) -> None:
             raise TypeError(f"expected type None or str, got {type(value)}")
 
     @property
-    def mark_for_deletion(self) -> dict:
+    def mark_for_deletion(self) -> MarkForDeletion:
```

Comment:
Could you rename this to `marked_for_deletion` in order to emphasize that it is not an action, to contrast it from `mark_deletion`?

Reply:
sure thing.